projects/lux/src/lib/datetime/datetime.component.scss

```diff
@@ -0,0 +1,77 @@
+@import '../styles/lux';
+
+$input-postfix-line-height: var(--lux-input-postfix-line-height, 1.5px);
+$input-postfix-font-size: var(--lux-input-postfix-font-size, 1rem);
+$input-postfix-height: var(--lux-input-postfix-height, inherit);
+
+:host {
+  position: relative;
+  display: flex;
+  align-items: stretch;
+  width: 20rem;
+}
+
+input {
+  margin: $input-margin;
+  padding: $input-padding;
+}
+
+.alert {
```

Comment:
CS Refactor: move the .alert definition to styles/lux.scss  (the .alert is defined three times in lux). DRY
Look for common/repeated CSS and define only once.

Reply:
Same for .symbol, prefix, postfix ->define only once if posible and import style/lux